ios/Classes/PurchasesFlutterPlugin.m

```diff
@@ -435,6 +438,11 @@ - (void)setCreative:(nullable NSString *)creative result:(FlutterResult)result
     result(nil);
 }
 
+- (void)canMakePayments:(NSArray <NSNumber *>*)features result:(FlutterResult)result
```

Comment:
we should name this one `canMakePaymentsWithFeatures:result:` to follow the obj-c practice of referencing the first param in the method name